sdk/lib/rtl/rangelist.c

```diff
@@ -572,14 +1582,20 @@ RtlInitializeRangeList(IN OUT PRTL_RANGE_LIST RangeList)
  */
 NTSTATUS
 NTAPI
-RtlInvertRangeList(OUT PRTL_RANGE_LIST InvertedRangeList,
-                   IN PRTL_RANGE_LIST RangeList)
+RtlInvertRangeList(
+    _Out_ PRTL_RANGE_LIST InvertedRangeList,
+    _In_ PRTL_RANGE_LIST RangeList)
 {
     PRTL_RANGE_ENTRY Previous;
     PRTL_RANGE_ENTRY Current;
     PLIST_ENTRY Entry;
     NTSTATUS Status;
 
+    DPRINT("RtlInvertRangeList: InvertedRangeList %p, RangeList %p\n", InvertedRangeList, RangeList);
+
+// FIXME None Reworked !
+ASSERT(FALSE);
```

Comment:
To be reworked for the final version of the PR.

Reply:
These two functions have never been called and most likely never will be called...
Therefore, they are not modified. They will be much more difficult to remake, and debugging is generally impossible. And they will take a lot of time, which can be spent on "live" functions.

Reply:
I just replaced the RTL_RANGE_ENTRY on PRTLP_RANGE_LIST_ENTRY in the RTLinvertRangeList() function.
This is enough to compile. DPRINT1S at the beginning will let us know if there is a call to these functions.